docs/book/src/developers/development.md

```diff
@@ -112,6 +112,19 @@ Makefile targets and scripts are offered to work with go modules:
 Your environment must have the Azure credentials as outlined in the [getting
 started prerequisites](../topics/getting-started.md#Prerequisites) section.
 
+Install [Tilt](https://docs.tilt.dev/install.html):
```

Comment:
What do you think about moving these above with the other install instructions under `### Base requirements`? 

You could either add them directly to the list, or have a separate paragraph for Tilt requirements

Reply:
yeah let's do a separate section `### Tilt requirements` after `### Base requirements`